src/c++/perf_analyzer/docs/examples/profile.py

```diff
@@ -87,15 +136,26 @@ def get_plot_filename(args, prompt_size):
     return filename
 
 
+def save_benchmark_results(args, profile_results):
+    for pr in profile_results:
+        postfix = get_postfix(args, pr.prompt_size)
+        results_csv = f"results-{postfix}.csv"
+        with open(results_csv, "w") as f:
+            fieldnames = [f.name for f in fields(pr)]
+            writer = csv.DictWriter(f, fieldnames=fieldnames)
```

Comment:
I'm hesitant with adding CSV output. It feels like it's entrenching us more and more into this script, when we really want to shift users to MA. I feel like the more structured we make the output, the more users will assume long-term support. Maybe we can just print all of this to stdout for now? Idk, it's a lot of numbers...

@tgerdesnv @debermudez @nv-braf any ideas?

Reply:
Yeah, if this is just intended to be some examples and easy charting, it feels like csv is overkill until there is a need for it

Reply:
@matthewkotila Good point. We can remove the csv output.